setup.py

```diff
@@ -62,6 +62,10 @@
     'cmake>=3.25.0,<=3.26.3',  # required for triton-pre-mlir below
     # PyPI does not support direct dependencies, so we remove this line before uploading from PyPI
     'triton-pre-mlir@git+https://github.com/vchiley/triton.git@triton_pre_mlir_sm90#subdirectory=python',
+    'loralib',  # lora
+    'peft',  # lora
+    'bitsandbytes',  # 8bit
+    'scipy'
```

Comment:
what is `scipy` for? also, let's pin down the versions to a range or value

Reply:
done; pinned down everything but peft which changes daily and we need their changes. 
`scipy` is a dep of `bitsandbytes` though it doesn't appear in their setup.py (raised an issue there).